index.js

```diff
@@ -5,6 +5,8 @@ import session from 'express-session';
 import cors from 'cors';
 import dotenv from 'dotenv';
 import errorhandler from 'errorhandler';
+// import swaggerUi from 'swagger-ui-express';
```

Comment:
#### Nitpick
Remove dead codes

Reply:
@KaiserPhemi this line was as a result of the code I rebased, chisom added support for swagger without swagger.yml file, so when I rebased my code was bringing error, so I had to comment the lines using swagger out in order to push